src/components/VerticalNavigation/VerticalNavigation.stories.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withKnobs } from '@storybook/addon-knobs';
+import { defaultTemplate } from '../../../storybook/decorators/storyTemplates';
+
+import { VerticalNavigation, VerticalNavigationItem } from '../../index';
+import { mockNavItems } from './__mocks__/mockNavItems';
+
+const stories = storiesOf('VerticalNavigation', module);
+stories.addDecorator(withKnobs);
+stories.addDecorator(
+  defaultTemplate({
+    title: 'VerticalNavigation',
+    documentationLink:
+      'http://www.patternfly.org/pattern-library/navigation/vertical-navigation/',
+  }),
+);
+
+stories.addWithInfo(
+  'Vertical Navigation driven by JSX children',
+  `VerticalNavigation usage example with items passed as children, but with only titles.`,
+  () => {
+    return (
+      <VerticalNavigation secondaryCollapsed={false} tertiaryCollapsed>
+        <VerticalNavigationItem title="Item 1" />
```

Comment:
This is a nit but can we expose the API as `VerticalNavigation.Item` ... see Wizard examples...it's just a consistent thing with the other complex components (both in PF React and React Bootstrap) where we have a lost of sub components.